src/components/TasksTable/index.jsx

```diff
@@ -284,19 +284,19 @@ export default class TasksTable extends Component {
 
     return (
       <Fragment>
-        <div className={classes.flexContainer}>
+        <Toolbar className={classes.toolbar}>
+          <Typography variant="title" className={classes.title}>
+            Bugs & Issues
+          </Typography>
+          <IconButton onClick={this.handleFilterToggle}>
+            <FilterVariantIcon />
+          </IconButton>
```

Comment:
Have the filter icon come after the "I'm Feeling Adventurous" button.

Reply:
ok, i initially display the button after the icon to make it nicer on mobile 
<img width="454" alt="screen shot 2018-07-19 at 1 22 33 pm" src="https://user-images.githubusercontent.com/24544912/42925278-d3013eca-8b57-11e8-8e82-c450285fcbc2.png"> but i guess the media query can do the same